var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -215,7 +220,7 @@ def cmake_args(self):
 
         options.extend(
             [
-                define(project_name_base + "_ENABLE_TESTS", True),
+                define(project_name_base + "_ENABLE_TESTS", from_variant("tests")),
```

Comment:
This could possibly be `from_variant(project_name_base + "_ENABLE_TESTS", "tests"),` but ok as is...